.circleci/config.yml

```diff
@@ -63,21 +63,24 @@ jobs:
     docker:
       - image: circleci/python:2.7.14-jessie
     environment:
-     - TOX_ENV: "py27,CI-py27"
+     - TEST_TOX_ENV: "py27,CI-py27"
+     - COVERAGE_TOX_ENV: "py27"
     <<: *ci-steps
 
   python35:
     docker:
       - image: circleci/python:3.5.4-jessie
     environment:
-     - TOX_ENV: "py35,CI-py35"
+     - TEST_TOX_ENV: "py35,CI-py35"
+     - COVERAGE_TOX_ENV: "py35"
```

Comment:
Are the variable switched ? Look like `COVERAGE_TOX_ENV` should be associated with `CI-py35`

Reply:
Also not clear why it is `py35,CI-py35` instead of `CI-py35`

Reply:
I think you are right. I think all envs inherit from testenv so CI-py35 should be enough.

Reply:
I think I will refactor tox.ini. We should not need comments there.